www/voll03/sp2/assets/js/localstorage.js

```diff
@@ -0,0 +1,103 @@
+import { API, DOM } from './resources.js';
+
+/**
+ * Check if movie with given id is in favourites
+ * @param {*} movie_id 
+ * @returns true/false 
+ */
+export const isFavourite = (movie_id) => {
+    const favList = JSON.parse(localStorage.getItem('favourites'));
+
+    if (favList.length > 0) {
+        const index = favList.findIndex(item => item.id === movie_id);
+        return (index !== -1);
+    }
+
+    return false;
+};
+
+/**
+ * Get user rating of movie with given id
+ * @param {*} movie_id 
+ * @returns 0   = movie is not rated - no rating found
+ *          int = movie rating 
+ */
+export const getRating = (movie_id) => {
+    const ratedList = JSON.parse(localStorage.getItem('rated'));
+
+    if (ratedList.length > 0) {
```

Comment:
zobrazeni detailu filmu pada tady, to same u rated a favorites

Reply:
Aha, tak to mi při testování uniklo, omlouvám se. Ve firefoxu mi aplikace normálně funguje, ale na jiných prohlížečích skutečně v tomto místě padá. Opravím zítra v rámci obhajoby.

Reply:
Opraveno